src/UI/IdentityBuilderUIExtensions.cs

```diff
@@ -44,63 +40,18 @@ public static IdentityBuilder AddDefaultUI(this IdentityBuilder builder)
 
         private static void AddAdditionalApplicationParts(IdentityBuilder builder)
```

Comment:
Ignore the version changes \ new package references etc that were made. The intent of the PR is to ensure the design of the Razor.Sdk, ApplicationPartFactory is as we need. I verified that the functional tests in the app work and we code gen the same set of attributes that we previously did.